src/DraggableArea/stories/DraggableArea.stories.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import uuidv1 from 'uuid/v1';
+import Modal from '../../Modal';
+import DragableArea from '../index';
+import ModalReadme from '../README.md';
+
+const styles = {
```

Comment:
We shouldn't have hardcoded styles in the components, we should use classNames instead, you could generate css just form demo proposes. If the styles are 100% necessaries to the component then we should think to include scss files to do this kind of exceptions. If isn't an exception and are not 100% necessary just delete it or moved to css form demo proposes.